src/components/plots/linePlot.ts

```diff
@@ -27,6 +27,16 @@ export module Plot {
       this._defaultStrokeColor = new Scale.Color().range()[0];
     }
 
+    public interpolate(): string | ((points: number[][]) => string);
```

Comment:
If you can find a way to only have the (points: number[][]) => string signature I would prefer that one, however this would require that we have a set of interpolators. (Is there a way we can pull the d3 interpolators into another file?)  If not, please specify that we are using d3's interpolators in the docs.